crates/turbopack-css/src/util.rs

```diff
@@ -0,0 +1,50 @@
+use lazy_static::lazy_static;
+use regex::Regex;
+
+pub fn stringify_str(str: &str) -> String {
+    lazy_static! {
+        static ref CSS_STR_ESCAPE_CHARS: Regex =
+        // Escape the following characters required for strings by the CSS scanner:
+        // https://www.w3.org/TR/CSS21/grammar.html#scanner
+        // https://github.com/vercel/turbo/pull/2598#discussion_r1022625909
+        //
+        // Note that the form feed character is not representable as \f in Rust strings, so
+        // the unicode representation \u{0c} is used.
+        regex::Regex::new("(\"|\n|\r|\\\\|\u{0c})").unwrap();
+    };
+
+    format!("\"{}\"", CSS_STR_ESCAPE_CHARS.replace_all(str, "\\$1"))
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::util::stringify_str;
+
+    #[test]
+    fn surrounds_with_double_quotes() {
+        assert_eq!(stringify_str("foo"), "\"foo\"");
+    }
+
+    #[test]
+    fn escapes_double_quotes() {
+        assert_eq!(stringify_str("\"\""), "\"\\\"\\\"\"");
+    }
+
+    #[test]
+    fn escapes_backslash() {
+        assert_eq!(stringify_str("\\\\"), "\"\\\\\\\\\"");
+    }
+
+    #[test]
+    fn escapes_newlines() {
+        assert_eq!(stringify_str("\n\n"), "\"\\\n\\\n\"");
```

Comment:
Add a test for escaped newlines: `\\n`, which would expose the double backslash bug.